README.md

```diff
@@ -1,27 +1,32 @@
-AncientWarfare2
+Ancient Warfare 2
 ===============
+
 In order to develop on this project you will need to:  
-Download the repository into a local folder.  
-Locate this folder, run gradle_update.bat  
-This should download all dependencies and setup proper .classpath  
-Import the project into eclipse from the downloaded folder/repository.  
+
+1) Download the repository into a local folder.  
+
+2) Locate this folder, run `gradlew.bat` or `./gradlew`.
```

Comment:
This should really be run gradlew setupDecompWorkspace

Reply:
`gradlew.bat setupDecompWorkspace`? I don't have Windows.

I'll update `./gradlew setupDecompWorkspace` and send a PR shortly.

Reply:
you don't need to type ".bat" on windows. 
And somehow I thought you don't need "./" on mac when runing in current forlder in which case the command would be gradlew setupDecompWorkspace on both, but if you actually need to type that in that's the only difference.